test/e2e/utils/test_context.go

```diff
@@ -243,10 +243,16 @@ func (t *TestContext) Destroy() {
 	//nolint:gosec
 	// if image name is not present or not provided skip execution of docker command
 	if t.ImageName != "" {
-		cmd := exec.Command("docker", "rmi", "-f", t.ImageName)
-		if _, err := t.Run(cmd); err != nil {
-			warnError(err)
+		// Check white space from image name
+		if len(strings.Trim(t.ImageName, "")) == 0 {
```

Comment:
This should either be:
```go
strings.Trim(t.ImageName, " ")
```
or

```go
strings.TrimSpace(t.ImageName)
```

First option has an empty space in the second argument of the trim, which means we are intending to remove all whitespaces. The second one is a helper provided by golang itself: https://pkg.go.dev/strings#TrimSpace